backend/src/api.py

```diff
@@ -528,3 +528,30 @@ def cleanup():
 @app.get("/stats")
 def get_report_stats() -> dict:
     return Storator3000.get_stats()
+
+
+@app.get("/contributions/{username}")
+def get_contributions(request: Request, username: str):
+    """
+    A list of contributions for a given FAS username
+    """
+    logs = Storator3000.get_logs()
+
+    # This is brutal ... but we don't have a database so there is no other way
```

Comment:
agreed, one day we have to move the data to an actual DB, but for now we need to do this